src/wrappers/themis/rust/libthemis-sys/build.rs

```diff
@@ -111,3 +124,37 @@ impl CCBuildEx for cc::Build {
         self
     }
 }
+
+fn try_system_themis() -> Option<Library> {
+    let mut build = cc::Build::new();
+    build.file("src/dummy.c");
+    build.flag("-lthemis");
+
+    match build.try_compile("dummy") {
+        Ok(_) => {
+            eprintln!(
+                "\
+`libthemis-sys` tried using standard system paths and it seems that Themis
```

Comment:
imho this message should be printed from `get_themis` function, not here. function `try_system_themis` just check is themis exists in themis and doesn't know anything about high-level concerns like `libthemis-sys` existence. here just return success/error code and maybe print how it was checked and why it is failed

Reply:
Makes sense. I'll move these error/status messages up the call stack.